### PR TITLE
👪 Clean up export code and include groups and fields

### PR DESCRIPTION
**Pull request by rowanseymour**
* Decouple export versions from flow spec versions
* Bump the current version to 13 and add checks so versions ahead of the instance version can't be imported
* Include groups and fields in new sections in the export (https://github.com/rapidpro/rapidpro/issues/937)

### CODE REVIEW

File: temba/campaigns/models.py [hunk 1]
```diff
@@ -191,10 +186,10 @@ def apply_action_restore(cls, user, campaigns):
     def get_events(self):
         return self.events.filter(is_active=True).order_by("relative_to", "offset")
 
-    def as_json(self):
+    def as_export_json(self):
```
Comment (rowanseymour) [inline temba/campaigns/models.py]: making this a consistent name for methods that generate the export representation

File: temba/contacts/models.py [hunk 2]
```diff
@@ -731,11 +731,6 @@ def as_search_json(self):
 
         return obj
 
-    def groups_as_text(self):
-        groups = self.user_groups.all().order_by("name")
-        groups_name_list = [group.name for group in groups]
-        return ", ".join(groups_name_list)
```
Comment (rowanseymour) [inline temba/contacts/models.py]: not used anywhere

File: temba/campaigns/models.py [hunk 3]
```diff
@@ -48,114 +48,109 @@ def get_unique_name(cls, org, base_name, ignore=None):
...
     @classmethod
-    def import_campaigns(cls, exported_json, org, user, same_site=False):
+    def import_campaigns(cls, org, user, campaigns_json, same_site=False):
         """
-        Import campaigns from our export file
+        Import campaigns from a list of exported campaigns
         """
-        from temba.orgs.models import EARLIEST_IMPORT_VERSION
-
-        if Flow.is_before_version(exported_json.get("version", "0"), EARLIEST_IMPORT_VERSION):  # pragma: needs cover
```
Comment (rowanseymour) [inline temba/campaigns/models.py]: this check happens once in `Org.import_app`

File: temba/contacts/models.py [hunk 4]
```diff
@@ -3023,9 +3029,36 @@ def release(self):
     def is_dynamic(self):
         return self.query is not None
 
-    def analytics_json(self):
```
Comment (rowanseymour) [inline temba/contacts/models.py]: not used anywhere

File: temba/campaigns/models.py [hunk 5]
```diff
@@ -221,8 +222,12 @@ def as_json(self):
 
             events.append(event_definition)
 
-        definition["events"] = events
-        return definition
+        return {
+            Campaign.EXPORT_UUID: str(self.uuid),
+            Campaign.EXPORT_NAME: self.name,
+            Campaign.EXPORT_GROUP: {"uuid": str(self.group.uuid), "name": self.group.name},
```
Comment (nicpottier) [inline temba/campaigns/models.py]: Should we introduce the idea of `as_reference()` as a method maybe?

**Pull request merged**